src/main/scala/coreplex/BaseCoreplex.scala

```diff
@@ -50,7 +50,7 @@ abstract class BaseCoreplex(c: CoreplexConfig)(implicit p: Parameters) extends L
 
 abstract class BaseCoreplexBundle(val c: CoreplexConfig)(implicit val p: Parameters) extends Bundle with HasCoreplexParameters {
   val master = new Bundle {
-    val mem = Vec(c.nMemChannels, new ClientUncachedTileLinkIO()(outermostParams))
+    val mem = Vec(c.nMemChannels, new ClientUncachedTileLinkIO()(outerMemParams))
```

Comment:
No, before the width adapter was still inside the coreplex. Now the width adapter is in the periphery. That way there are no edge* parameters in the coreplex. I think that's the most sensible and consistent way.